PW39_2023_Montreal/README.md

```diff
@@ -52,6 +52,40 @@ Venue entrance on Google Maps: [https://goo.gl/maps/xNedgMBt4C6jwiCu5](https://g
 
 ## Contributing Project Pages
 
+### Videos in project pages
+
+Here are some steps to make sure all of your awesome videos render correctly:
+
+1. Videos added by drag and drop will render correctly when viewed through GitHub, but need some extra tweaks to work in the final generated webiste.  In your README.md, if you have a video link that looks like this:
+    ````
+    https://github.com/NA-MIC/ProjectWeek/assets/66890913/8f257f29-fa9c-4319-8c49-4138003eba27
+    ````
+
+    Update it to:
+    ````
+    <video
+      controls muted
+      src="https://github.com/NA-MIC/ProjectWeek/assets/66890913/8f257f29-fa9c-4319-8c49-4138003eba27"
+      style="max-height:640px; min-height: 200px">
+
+    </video>
```

Comment:
Good catch.

Totally missed that only specifying `/>` to close the tag was causing the remaining of the document was then added as content of the video tag.

![image](https://github.com/NA-MIC/ProjectWeek/assets/219043/7ba938c2-3cb4-487a-86f7-df42f38f1e59)